wordpress-mobile/WordPress-Android
# Feature/15609 add tracks

Fixes #15609 

This PR updates with necessary changes to allow posting tracks.  It depends on [Part 1](https://github.com/Automattic/about-automattic-android/pull/18)  PR and [fixes](https://github.com/Automattic/about-automattic-android/pull/19) on about-automattic-android library

NOTE:  After merging [Part1](https://github.com/Automattic/about-automattic-android/pull/18).  Need to update the about-automattic-library version in the build.gradle


To test:

**Setup: Enable feature flag**

1. On the Main screen, tap the Me button.
1. On the Me screen, go to App Settings > Debug settings.
1. On the Debug Settings screen, scroll down to the "Features in development" section and tap `UnifiedAboutFeatureConfig` to enable the About feature flag.
1. Restart the app

**Test**

1. On the Main screen, tap the Me button.
1. Notice About WordPress option
1. Tap on the About WordPress item.
1. Click on each of the items in order and
1. Notice in the Logcat the following tracks are logged


```
🔵 Tracked: about_screen_shown, Properties: {"screen":"main"}
🔵 Tracked: about_screen_button_tapped, Properties: {"button":"share"}
🔵 Tracked: about_screen_button_tapped, Properties: {"button":"rate_us"}
🔵 Tracked: about_screen_button_tapped, Properties: {"button":"twitter"}

🔵 Tracked: about_screen_button_tapped, Properties: {"button":"legal"}
🔵 Tracked: about_screen_dismissed, Properties: {"screen":"legal_and_more"}
🔵 Tracked: about_screen_button_tapped, Properties: {"button":"terms_of_service"}
🔵 Tracked: about_screen_button_tapped, Properties: {"button":"privacy_policy"}
🔵 Tracked: about_screen_button_tapped, Properties: {"button":"acknowledgements"}
🔵 Tracked: about_screen_dismissed, Properties: {"screen":"legal_and_more"}

🔵 Tracked: about_screen_button_tapped, Properties: {"button":"automattic_family"}
🔵 Tracked: about_screen_button_tapped, Properties: {"button":"work_with_us"}

🔵 Tracked: about_screen_dismissed, Properties: {"screen":"main"}

```

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
Manual testing

PR submission checklist:
